### ISSUE TITLE
Using Symmetric array destructuring throws errors in phpmd

### ISSUE THREAD

**Issue by chadidi** [phpmd/phpmd#728]
- PHPMD version: 2.7
- PHP Version: 7.3
- Installation type: composer
- Operating System / Distribution & Version: (ubuntu server) on mac os machine

I get: `Avoid using undefined variables such as '$latitude' which will lead to PHP notices.`

### Expected Behavior
no such error.

### Steps To Reproduce:
```
[$latitude, $longitude] = ['35.567211', '-5.371620'];
# then try to use them!

**Comment by kylekatarnls** [comment]
Thanks. I close as this is a duplicate of #725

**Comment by kylekatarnls** [comment]
This should be fixed with #770 and you can yet test it requiring `"phpmd/phpmd": "dev-master"`
